### ISSUE TITLE
`import_user': undefined method `returning' on snapshot import

### ISSUE THREAD

**Issue by meleyal** [wesabe/pfc#21]
I get the following error when trying to import the snapshot with thor (0.14.0):

```
$ bundle exec thor snapshot:import ~/Downloads/wesabe-data/htxt.zip 
unzipping snapshot...
importing user...
/Applications/wesabe/pfc/lib/importer/wesabe.rb:92:in `import_user': undefined method `returning' for #<Importer::Wesabe:0x10331f1b0> (NoMethodError)
    from /Applications/wesabe/pfc/lib/importer/wesabe.rb:36:in `import'
    from /Applications/wesabe/pfc/lib/importer/wesabe.rb:12:in `import'
    from /Applications/wesabe/pfc/lib/tasks/snapshot.thor:10:in `import'
    from /Users/william/.rvm/gems/ruby-1.8.7-p299/gems/thor-0.14.0/lib/thor/task.rb:22:in `send'
    from /Users/william/.rvm/gems/ruby-1.8.7-p299/gems/thor-0.14.0/lib/thor/task.rb:22:in `run'
    from /Users/william/.rvm/gems/ruby-1.8.7-p299/gems/thor-0.14.0/lib/thor/invocation.rb:118:in `invoke_task'
    from /Users/william/.rvm/gems/ruby-1.8.7-p299/gems/thor-0.14.0/lib/thor.rb:246:in `dispatch'
    from /Users/william/.rvm/gems/ruby-1.8.7-p299/gems/thor-0.14.0/lib/thor/base.rb:389:in `start'
    from /Users/william/.rvm/gems/ruby-1.8.7-p299/gems/thor-0.14.0/lib/thor/runner.rb:34:in `method_missing'
    from /Users/william/.rvm/gems/ruby-1.8.7-p299/gems/thor-0.14.0/lib/thor/task.rb:22:in `send'
    from /Users/william/.rvm/gems/ruby-1.8.7-p299/gems/thor-0.14.0/lib/thor/task.rb:22:in `run'
    from /Users/william/.rvm/gems/ruby-1.8.7-p299/gems/thor-0.14.0/lib/thor/task.rb:108:in `run'
    from /Users/william/.rvm/gems/ruby-1.8.7-p299/gems/thor-0.14.0/lib/thor/invocation.rb:118:in `invoke_task'
    from /Users/william/.rvm/gems/ruby-1.8.7-p299/gems/thor-0.14.0/lib/thor.rb:246:in `dispatch'
    from /Users/william/.rvm/gems/ruby-1.8.7-p299/gems/thor-0.14.0/lib/thor/base.rb:389:in `start'
    from /Users/william/.rvm/gems/ruby-1.8.7-p299/gems/thor-0.14.0/bin/thor:6
    from /Users/william/.rvm/gems/ruby-1.8.7-p299/bin/thor:19:in `load'
    from /Users/william/.rvm/gems/ruby-1.8.7-p299/bin/thor:19
```

**Comment by meleyal** [comment]
It seems Object#returning was removed from Rails 3.0:
http://github.com/rails/rails/commit/b0b9bf320409b66c6c6b680371aca590297cd4cc

And is now back again:
http://github.com/rails/rails/commit/88b5f938cf7d3eb26ad204451a4dbb9c2cf4f571

**Comment by eventualbuddha** [comment]
Replace `returning` with `tap`. Closed by 1156004e422aa68dc1ec5e7b9e3a9bb1cedb4645.
